AuroraAssetEditor/Controls/OnlineAssetsControl.xaml.cs

```diff
@@ -12,19 +12,27 @@ namespace AuroraAssetEditor.Controls {
     using System.Drawing.Imaging;
     using System.Globalization;
     using System.IO;
-    using System.Text.RegularExpressions;
+	using System.Text.RegularExpressions;
     using System.Windows;
     using System.Windows.Controls;
     using System.Windows.Input;
     using System.Windows.Media.Imaging;
-    using Classes;
-    using Image = System.Drawing.Image;
+	using AuroraAssetEditor.Helpers;
+	using Classes;
+	using System.Linq;
+	using Image = System.Drawing.Image;
 
     /// <summary>
     ///     Interaction logic for OnlineAssetsControl.xaml
     /// </summary>
     public partial class OnlineAssetsControl {
-        private readonly BackgroundControl _background;
+		private enum SelectedItem
```

Comment:
The name of this enum should be something that explains what it is, "SelectedItem" is a terrible name as it suggests it's a property, a better name would be "OnlineAssetSources" or something like that.